rlclientlib/model_mgmt/restapi_data_transport.cc

```diff
@@ -38,50 +41,62 @@ namespace reinforcement_learning { namespace model_management {
    * x-ms-version = 2017-04-17
    */
 
-  int restapi_data_transport::get_data_info(::utility::datetime& last_modified, ::utility::size64_t& sz, api_status* status) {
+  int restapi_data_transport::get_data_info(::utility::datetime& last_modified, ::utility::size64_t& sz, api_status* status, http_request request) {
 
-    // Build request URI and start the request.
-    auto request_task = _httpcli->request(methods::HEAD)
-      // Handle response headers arriving.
-      .then([&](http_response response) {
-      if ( response.status_code() != 200 )
-        RETURN_ERROR_ARG(_trace, status, http_bad_status_code, _httpcli->get_url());
+      // Get request URI and start the request.
+      request.headers() = _header;
+      // Build request URI and start the request.
+      auto request_task = _httpcli->request(request).then([&](http_response response) {
+          if ( response.status_code() != 200 )
+              RETURN_ERROR_ARG(_trace, status, http_bad_status_code, _httpcli->get_url());
 
-      const auto iter = response.headers().find(U("Last-Modified"));
-      if ( iter == response.headers().end() )
-        RETURN_ERROR_ARG(_trace, status, last_modified_not_found, _httpcli->get_url());
+          const auto iter = response.headers().find(U("Last-Modified"));

```

Comment:
Can we define a constant string for "Last-Modified"?

Reply:
ried doing it but trying to do config.get to fetch constant throws error in headers.add and header().find